docs/pages/repo/docs/core-concepts/monorepos/skipping-tasks.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: Skipping tasks
+description: Never do the same work twice.
+---
+
+import Callout from '../../../../../components/Callout'
+
+# Skipping Tasks
+
+[Build caching](/repo/docs/core-concepts/caching) can dramatically speed up your build tasks - but you can do even better by using `npx turbo-ignore`. If your workspace is unaffected by your code changes, you can completely skip executing a task altogether.
+
+## Using `turbo-ignore`
+
+To skip unaffected work, ensure that your git history is available and run `npx turbo-ignore`.
+
+By default, `turbo-ignore` will run the **build** task in the current working directory. Under the hood, `turbo-ignore` uses the output of `--dry=json` from your parent commit to your current commit to identify affected packages.
+
+Here's an example of the command that will be built and ran:
+
+```bash
+npx turbo run build --filter=@example/web...[3c8387ffd98b751305fe3f0284befdd00cbd4610] --dry=json
+```
+
+Note that this command does not _execute_ the build task. Instead, it uses a **dry run** to see if your code changes will affect your build compared to your parent commit.
+
+If `turbo-ignore` finds that the task can be skipped, it will exit the process with a `0` code. If changes have been found, the process will exit with `1`.
+
+<Callout type="info">On Vercel, the previously deployed SHA will be used instead of the parent commit.</Callout>
+
+## Customizing behavior
+
+To specify a workspace, you can add it to your command like:
+
+```
+npx turbo-ignore web
+```
+
+where `web` is your workspace's name.
+
+If you'd like to change more of the default behavior, there are a few flags available:
+
+- `--task`: Specifies the task for the command that `turbo-ignore` will invoke. Defaults to `build`.
+- `--directory`: Specify a directory where `turbo-ignore` will run. Defaults to the current working directory.
+- `--fallback`: Specify a ref/head to compare against. Defaults to `HEAD^`.
+
+## Example
+
+Let's say you want to skip the unit tests for your `web` workspace when there aren't any changes to your `web` application or its package dependencies. In practice, hitting the cache with a `>>> FULL TURBO` will speed things up quite a bit but there will still be time spent installing `npm` dependencies, creating upcoming CI infrastructure, or doing other types of work that can take awhile.
+
+Ideally, we would do a quick check to see if any of that work needs to happen in the first place.
+
+After we've checked out the repo but **before** any other work, we can take a few seconds to check that our `web` tests have changed since the parent commit.
```

Comment:
```suggestion
After we've checked out the repo, but **before** any other work, we can take a few seconds to check that our `web` tests have changed since the parent commit.
```